staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -153,13 +153,20 @@ func (s *store) Get(ctx context.Context, key string, opts storage.GetOptions, ou
 
 // Create implements storage.Interface.Create.
 func (s *store) Create(ctx context.Context, key string, obj, out runtime.Object, ttl uint64) error {
+	trace := utiltrace.New("Create",
+		utiltrace.Field{"key", key},
```

Comment:
Do we need key?

If so - should we modify update too?

Reply:
Done